data/interfaces.txt

```diff
@@ -123,8 +123,8 @@ color "map name" .6 .6 .6 .6
 color "map travel ok fleet" .2 .5 0. 0.
 color "map travel ok flagship" .5 .4 0. 0.
 color "map travel ok none" .55 .1 0. 0.
-color "map used wormhole" .5 .2 .9 1.
-color "map unused wormhole" .165 .066 .3 .333
+color "map travel wormhole" .5 .2 .9 1.
+color "map wormhole" .5 .2 .9 1.
```

Comment:
The "map unused wormhole" colour is removed and doesn't appear to ever be reused?